src/index.js

```diff
@@ -1,10 +1,10 @@
 import { useEffect, useState, useRef } from "react";
 import ResizeObserver from "resize-observer-polyfill";
 
-export default function() {
+export default function(defaultWidth = 1, defaultHeight = 1) {
```

Comment:
I'd suggest defaulting the default (😆) to `0` so that we can use `!!width` to test if we've got a width.

Reply:
It could be that width / height is actually zero though.
Could return nulls, but I'm not sure it's worth the effort, and it's a breaking change as well.

Do you have a good use-case you'd need this for? @FokkeZB

Reply:
Ah, yes... then only `null` would work. I don't have a direct need for it right now, but I do think it would make sense to tell the default from an actual value.

Reply:
Feel free to create a separate issue when you have a use-case for it and then we can discuss it. 👍